composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "Apache-2.0",
     "require": {
         "php": "^7.2",
```

Comment:
This should be bumped to `7.3`, right?

Reply:
Yup; this was just missed when I was porting this over.  Will fix.